security/subsystem/src/main/java/org/jboss/as/security/AuditResourceDefinition.java

```diff
@@ -54,7 +54,7 @@ public void registerAttributes(final ManagementResourceRegistration resourceRegi
     @Override
     public void registerChildren(ManagementResourceRegistration resourceRegistration) {
         super.registerChildren(resourceRegistration);
-        resourceRegistration.registerSubModel(new MappingProviderModuleDefinition(Constants.PROVIDER_MODULE));
```

Comment:
> What happened to MappingProviderResourceDefinition? From what I see this is the only place it is used but unless github is not showing correctly it has not been removed.

Good point. I thought, `MappingProviderResourceDefinition` was still a parrent for some subclass, but now I see I mixed up `MappingProviderResourceDefinition` with `MappingModuleDefinition`. Let me have a look if the removal of `MappingProviderResourceDefinition` is the best option.